submit/templates/submit/file_process.html

```diff
@@ -98,9 +126,16 @@ <h2 class="is-size-4">Process Files</h2>
           <h3>TeXLive Compiler Summary</h3>
         </div>
         <div class="level-right">
-          <a class="button">View complete log</a>
+          <a class="button" href="{{ url_for('ui.compilation_log', submission_id=submission_id) }}" target="_blank">
+            View raw log
+          </a>
         </div>
       </div>
+      <!-- TODO: don't hard-code this. -->
```

Comment:
https://arxiv-org.atlassian.net/browse/ARXIVNG-1781 written to remember this

Reply:
Thanks!